### PR TITLE
Dsg dev

### PR DESCRIPTION
**Pull request by dunkgray**

### CODE REVIEW

File: scripts/examples/go_reprocess.sh [hunk 1]
```diff
@@ -6,17 +6,18 @@ if [[ "$HOSTNAME" == *"gadi"* ]]; then
...
-	module load ard-scene-select-py3-dea/dev_20230511
+	#module load ard-scene-select-py3-dea/dev_20230512
+	module load ard-scene-select-py3-dea/20230515
 
 fi
 
 # Run the local scene select
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 SSPATH="$DIR"/../..
```
Comment (gordonyeong) [inline scripts/examples/go_reprocess.sh]: these two seem to do the same thing?
`
[gy5636@gadi-login-07 gy5636]$ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
[gy5636@gadi-login-07 gy5636]$ echo $DIR
/g/data/u46/users/gy5636
[gy5636@gadi-login-07 gy5636]$ SSPATH="$DIR"/../..
[gy5636@gadi-login-07 gy5636]$ echo $SSPATH
/g/data/u46/users/gy5636/../..
[gy5636@gadi-login-07 gy5636]$ `

Comment (dunkgray) [reply scripts/examples/go_reprocess.sh]: /g/data/u46/users/gy5636 and
/g/data/u46/users/gy5636/../.. 
are different though.
if you do an ls on both paths you'll get different results.

Comment (gordonyeong) [reply scripts/examples/go_reprocess.sh]: yes i did... they are the same.

I took a screenshot but could not upload it.


Pasting it here.
`[gy5636@gadi-login-04 ~]$ cd /g/data/u46/users/gy5636/../..
[gy5636@gadi-login-04 u46]$ pwd
/g/data/u46
[gy5636@gadi-login-04 u46]$ ls
2015-Exposed_Earth_Mosaic_Project  ABS      ARG25-minerals  ck9838  DLCDv2  jps547        pjt554    public  temp                                 time-spectral-profiles-V1-batch3.py  users   wofs
2019geomedian                      agdc-v2  BHMAR           crcsi   dxr251  LATEST_PIXEL  products  S2-NRT  time-spectral-profiles-V1-batch2.py  tobeDeleted                          vmn547
[gy5636@gadi-login-04 u46]$ ls -la /g/data/u46/users/gy5636/../..
total 172
drwxrwsr-x+ 32 root   u46   4096 Feb 21 13:12 .
dr-xr-xr-x  67 root   root  1340 May 22 12:01 ..
drwxrws---+  4 vmn547 u46   4096 Jun  7  2016 2015-Exposed_Earth_Mosaic_Project
drwxr-sr-x+  3 brl654 u46   4096 Feb 24  2020 2019geomedian
drwxrws---+  7 tty547 u46   4096 Sep  2  2016 ABS
drwxrws---+  2 dg6911 u46   4096 Aug 12  2018 agdc-v2
drwxrws---+  9 wxw554 u46   4096 Feb  8  2017 ARG25-minerals
drwxrws---+  8 adh547 u46  20480 Mar 19  2016 BHMAR
drwxrwsr-x+  3 ck9738 u46   4096 Feb 21 13:12 ck9838
drwxrwsrwx+  7 sjo547 u46   4096 Jan 21  2016 crcsi
drwxrwsrwx+  7 dg6911 u46   4096 Nov 28  2017 DLCDv2
drwxrwsr-x+  2 dxr251 u46   4096 Dec 19  2019 dxr251
drwxr-sr-x+  4 jps547 u46   4096 Apr  9  2020 jps547
drwxrwsrwx+  2 dg6911 u46   4096 Aug 12  2018 LATEST_PIXEL
drwxrwsr-x+ 83 pjt554 u46  12288 Apr 14  2022 pjt554
drwxrwsr-x+  5 sjo547 u46   4096 Sep  2  2015 products
drwxrwsr-x+  6 wxw554 u46   4096 Aug 15  2018 public
drwxrwsr-x+  2 jps547 u46   4096 Nov  6  2020 S2-NRT
drwxrwsrwx+  6 lww554 u46   4096 Mar 31 16:41 temp
-rwxrwxrwx+  1 lxl554 u46  11500 Sep  2  2015 time-spectral-profiles-V1-batch2.py
-rwxrwxrwx+  1 lxl554 u46  10729 Sep  2  2015 time-spectral-profiles-V1-batch3.py
drwxrwsr-x+  2 sao547 u46   4096 Jun 11  2019 tobeDeleted
drwxrws---+  5 adh547 u46   4096 Jul 27  2016 .Trash-10169
drwx--S---+  5 ext547 u46   4096 Mar  6  2018 .Trash-11095
drwx--S---+  5 ck9738 u46   4096 Apr 20  2018 .Trash-11460
drwx--S---+  5 bt2744 u46   4096 Feb 19  2020 .Trash-12030
drwx--S---+  5 cb3058 u46   4096 Jul  9  2018 .Trash-12852
drwx--S---+  4 ah3144 u46   4096 Mar 28  2018 .Trash-12861
drwxrws---+  5 brl654 u46   4096 Apr  5  2018 .Trash-7200
drwxrws---+  4 alm547 u46   4096 Sep 28  2016 .Trash-7449
drwx--S---+  5 nrm547 u46   4096 Oct 12  2017 .Trash-8243
drwxrws---+ 59 wxw554 u46   4096 Jan 24 11:59 users
drwxrws---+ 10 vmn547 u46   4096 Jan 13  2021 vmn547
drwxrwsr-x+ 13 dg6911 u46   4096 Jan 21  2019 wofs
[gy5636@gadi-login-04 u46]$ 
`

Comment (gordonyeong) [reply scripts/examples/go_reprocess.sh]: Screenshot 2023-05-22 at 12 39 20 pm

Comment (gordonyeong) [reply scripts/examples/go_reprocess.sh]: yes.

**Review by gordonyeong — APPROVED** [review]

**Pull request merged**